tests/variable_scalar_test.py

```diff
@@ -89,7 +89,45 @@ def test_scalar_Variable_conversion_to_builtin_float_with_variance():
         float(var)
 
 
-def test_scalar_Variable_conversion_to_builtin_float_fails_with_array():
+def test_conversion_to_builtin_float_fails_with_array():
     var = sc.array(dims=['x'], values=[1.0])
     with pytest.raises(sc.DimensionError):
         float(var)
+
+
+def test_scalar_Variable_conversion_to_builtin_bool_True():
+    assert sc.scalar(True)
+
+
+def test_scalar_Variable_conversion_to_builtin_bool_False():
+    assert not sc.scalar(False)
+
+
+def test_scalar_Variable_conversion_to_builtin_bool_bad_dtype():
+    var = sc.scalar(value=1.0, unit=None)
```

Comment:
Out of interest, do we support conversion from integer? e.g. `sc.scalar(value=1, unit=None)`